src/handlers/distance.js

```diff
@@ -14,7 +14,7 @@ export function Distance() {
 		attr: 'dist',
 		unit: distance.label,
 		decimals: 5,
-		pointToAttr: (_, i) => (i > 0 ? this._data[i - 1].dist : 0) + (this._data[i].latlng.distanceTo(this._data[i > 0 ? i - 1 : i].latlng) * opts.distanceFactor) / 1000, // convert back km to meters
+		pointToAttr: (_, i) => (i > 1 ? this._data[i - 1].dist : 0) + (this._data[i].latlng.distanceTo(this._data[i > 1 ? i - 1 : i].latlng) * opts.distanceFactor) / 1000, // convert back km to meters
```

Comment:
@hupe13 it would be preferable to find a solution that does not indiscriminately "delete" any valid values (ie. `i ==0`)

This too would require some sort of ad-hoc documentation, written like this it is not easy to understand reason..

Reply:
The first value must be ignored because speed and pace in the tooltip of chart and marker at the beginning (the first point) is NaN.